specs/engage-digital_openapi3.yaml

```
@@ -1343,9 +1343,9 @@ paths:
           schema:
             type: string
         - description: >-
-            For an email or SMS source. For an email, an array containing the
+            For an email or SMS and WhatsApp source. For an email, an array containing the
             email addresses used in the “To” section of the email. This
-            parameter is mandatory when initiating a discussion. For a SMS, the
+            parameter is mandatory when initiating a discussion. For a WhatsApp message or a SMS, the
             number the SMS will be sent to. It must start with 00 or +, example:

```

let's use a more generic word so that it make sense for WhatsApp as well
```suggestion
            number the message will be sent to. It must start with 00 or +, example:
```